### PR TITLE
Docs tcp connect

### PR DESCRIPTION
**Pull request by Nadine2016**
# Description

Please include a high level summary of the changes.

This bug fixes ... \ This new feature can be used to ...

_Fill out any of the following sections that are relevant and remove the others_
## API changes
- Added x field to y resource
- ...

## Code changes
- Fix error in `Foo()` function
- Add `Bar()` function
- ...

## CI changes
- Adjusted schedule for x job
- ...

## Docs changes
- Added guide about feature x to public docs
- Updated README to account for y behavior
- ...

# Context

Users ran into this bug doing ... \ Users needed this feature to ...

See slack conversation [here](https://solo-io-corp.slack.com/archives/some/post)

## Interesting decisions
 
We chose to do things this way because ...

## Testing steps

I manually verified behavior by ...

## Notes for reviewers

Be sure to verify intended behavior by ...

Please proofread comments on ...

This is a complex PR and may require a huddle to discuss ...

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: docs/content/guides/security/limit-connection.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+   {{% /tab %}}
+   {{< /tabs >}}
+
+2. Verify that an upstream was automatically created for the echo service.
+   ```sh
+   kubectl get upstreams default-tcp-echo-1025 -n gloo-system
+   ```
+
+3. Create a TCP gateway with connection limit settings. The following gateway accepts only one active connection at any given time. Before closing a new connection, the gateway waits 2 seconds. 
+   ```
```
Comment (artberger) [inline docs/content/guides/security/limit-connection.md:70]:
```suggestion
   ```yaml
```

File: docs/content/guides/security/limit-connection.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+      ...
+      - name: tcp
+        nodePort: 30197
+        port: 8000
+        protocol: TCP
+        targetPort: 8000
+      ```
+
+      Your `spec.ports` section looks similar to the following:
+      ```
```
Comment (artberger) [inline docs/content/guides/security/limit-connection.md:112]:
```suggestion
      ```yaml
```

File: docs/content/guides/security/limit-connection.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+        port: 443
+        protocol: TCP
+        targetPort: 8443
+      - name: tcp
+        nodePort: 30197
+        port: 8000
+        protocol: TCP
+      ```
+
+5. Get the public IP address of your gateway proxy. Note that the following commands returns the IP address and the default port.
```
Comment (artberger) [inline docs/content/guides/security/limit-connection.md:130]:
```suggestion
5. Get the public IP address of your gateway proxy. Note that the following command returns the IP address and the default port. 
```

File: docs/content/guides/security/limit-connection.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+   telnet <public-gateway-IP> 8000
+   ```
+
+   Example output:
+   ```
+   Connected to 113.21.184.35.bc.googleusercontent.com.
+   Escape character is '^]'.
+   ```
+
+8. Enter any string and verify that the echo service returns the same string. For example, you can type in `hello`.
```
Comment (artberger) [inline docs/content/guides/security/limit-connection.md:146]:
```suggestion
8. Enter any string and verify that the echo service returns the same string. For example, you can enter `hello`.
```

File: docs/content/guides/security/limit-connection.md [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+     labels:
+       gloo: tcp-echo
+     name: tcp-echo
+   spec:
+     containers:
+     - image: soloio/tcp-echo:latest
+       imagePullPolicy: IfNotPresent
+       name: tcp-echo
+     restartPolicy: Always
+  ---
```
[the remark this answers is not in the document]
Comment (Nadine2016) [reply docs/content/guides/security/limit-connection.md:38]: yes it looked good in the local build.

**Review by artberger — APPROVED** [review]

File: docs/content/guides/security/limit-connection.md [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+      ```
+
+5. Get the public IP address of your gateway proxy. Note that the following command returns the IP address and the default port. 
+   ```sh
+   glooctl proxy address
+   ```
+
+6. Open a telnet session to the public IP address of the gateway and port 8000.
+   ```sh
+   telnet <public-gateway-IP> 8000
```
Comment (davidjumani) [inline docs/content/guides/security/limit-connection.md:137]: Should this be `30197` ?

File: docs/content/guides/security/limit-connection.md [hunk 7]
```diff
@@ -0,0 +1,188 @@
...
+     tcpGateway:
+       options:
+         connectionLimit:
+           maxActiveConnections: 1
+           delayBeforeClose: 2s
+       tcpHosts:
+       - name: one
+         destination:
+           single:
+            upstream:
```
Comment (davidjumani) [inline docs/content/guides/security/limit-connection.md:89]: Mismatched spacing
```suggestion
             upstream:
```

File: docs/content/guides/security/limit-connection.md (hunk 6 above)
Comment (davidjumani) [reply docs/content/guides/security/limit-connection.md:137]: Works on 8000, nevermind

**Review by davidjumani — APPROVED** [review]

**Review by artberger — APPROVED** [review]

**Review by davidjumani — APPROVED** [review]

**Pull request merged**